website/content/en/preview/getting-started/_index.md

```diff
@@ -40,43 +40,22 @@ authenticate properly by running `aws sts get-caller-identity`.
 
 ### Environment Variables
 
-After setting up the tools, set the following environment variables to store
-commonly used values.
+After setting up the tools, set the following environment variable to the Karpenter version you
+would like to install.
 
 ```bash
-export CLUSTER_NAME="${USER}-karpenter-demo"
-export AWS_DEFAULT_REGION="us-west-2"
-export AWS_ACCOUNT_ID="$(aws sts get-caller-identity --query Account --output text)"
+export KARPENTER_VERSION={{< param "latest_release_version" >}}
```

Comment:
I think latest_release_version is wrong. If you look at an old version of the docs, this value will be wrong.

Reply:
this value is fixed for old versions from here: https://github.com/aws/karpenter/blob/main/hack/release.sh#L40 

I think we could make this more of a relative ref to the version and get rid of the release.sh hack :D

Reply:
Ah got it :D.

There are parts of the code base that get this from the path.

Reply:
@ellistarn Is this what you are referring to?
https://github.com/aws/karpenter/blob/7bc80906acc6d182216ed8d8a816803446d960d9/website/content/en/preview/getting-started/_index.md?plain=1#L126
If so that thing returns more than just a version, AFAICT it also includes `getting-started/` as well, fortunately latest_release_version happens works :)